packages/fetchai/skills/erc1155_deploy/README.md

```diff
@@ -0,0 +1,23 @@
+# ERC1155 Client
+
+## Description
+
+This is a skill for selling data via a smart contract.
+
+This skill registers some data selling service on the soef. It can be requested (for example by an agent with the generic_buyer skill) to provide specific data. It then negotiates the price and delivers the data after it receives payment.
```

Comment:
```suggestion
This skill registers some data selling service on the SOEF. It can be requested (for example by an agent with the generic_buyer skill) to provide specific data. It then negotiates the price and delivers the data after it receives payment.
```